config.go

```diff
@@ -265,7 +266,8 @@ func GetConfig(ctx context.Context) (*Settings, error) {
 
 // GetSharedConfig returns only the Evergreen configuration which is shared among all instances
 // reading from a single shared database. Use [GetConfig] to get a complete configuration that
-// includes overrides from the local database.
+// includes overrides from the local database. If there is no shared database [GetConfig] and
```

Comment:
I can't comment on lines outside the changes, but in the godoc for `GetConfig` it says use `GetSharedConfig` for *only* shared configuration. Could you update it to say something similar to this comment?